xbmc/cores/DllLoader/exports/emu_msvcrt.cpp

```diff
@@ -1819,16 +1819,18 @@ extern "C"
 
       if (value_start != NULL)
       {
-        char var[64];
+        const size_t varSize = value_start - envstring;
+        char* var = static_cast<char*>(std::malloc(varSize + 1));
+
```

Comment:
I would prefer to see the check if ` malloc` succeeded directly after the allocation. The way it is currently it would leak memory if only one `malloc` failed. Not really a big issue in practice but something static analyzers will complain about. 
```suggestion
        const size_t varSize = value_start - envstring;
        char* var = static_cast<char*>(std::malloc(varSize + 1));

        if (!var)
          return -1;
```